README.md

```diff
@@ -208,6 +283,32 @@ A Loop In swap can be executed a follows:
 ```
 loop in <amt_in_satoshis>
 ```
+
+#### Fees explained
+
+The following is an example output of a 0.01 BTC Loop In swap from `testnet`:
+
+```bash
+$ loop in --amt 1000000
+Max swap fees for 1000000 sat Loop In: 1562 sat
+CONTINUE SWAP? (y/n), expand fee detail (x): x
+
+Estimated on-chain HTLC fee:         154 sat
+Max swap fee:                        1100 sat
+CONTINUE SWAP? (y/n):
+```
+
+Explanation:
+
+- **Estimated on-chain HTLC fee** (154 sat): The estimated on-chain fee that the
+  daemon has to pay to publish the HTLC. This is an estimation from `lnd`'s
+  wallet based on the available UTXOs and current network fees. This value is
+  called `miner_fee` in the gRPC/REST responses.
+- **Max swap fee** (110 sat): The maximum amount of service fees the server is
+  going to charge for executing the swap. The actual fees might be lower if user
```

Comment:
Also here this is the maximum that we accept as a client. If the server proposes a higher amount, we don't publish the htlc and cancel the swap.